### ISSUE TITLE
error in 2d_rayleigh_benard.py

### ISSUE THREAD

**Issue by francispoulin** [DedalusProject/dedalus#78]
I am brand new to dedaus and have just installed it using conda.  The tests did no work but most of the examples do. 

One of the examples that does no wokr is 2d_rayleigh_benard.py  when I try running it I get the following error.  Could it be that this script needs to be updated?  Or my conda needs to be updated?  Any help would be greatly appreciated.  

```
    $ python rayleigh_benard.py 
    2020-06-18 10:22:32,757 pencil 0/1 INFO :: Building pencil matrix 1/128 (~1%) Elapsed: 0s, 
    Remaining: 3s, Rate: 4.9e+01/s
    2020-06-18 10:22:32,978 pencil 0/1 INFO :: Building pencil matrix 13/128 (~10%) Elapsed: 0s, 
    Remaining: 2s, Rate: 5.4e+01/s
    2020-06-18 10:22:33,218 pencil 0/1 INFO :: Building pencil matrix 26/128 (~20%) Elapsed: 0s, 
    Remaining: 2s, Rate: 5.4e+01/s
    2020-06-18 10:22:33,449 pencil 0/1 INFO :: Building pencil matrix 39/128 (~30%) Elapsed: 1s, 
    Remaining: 2s, Rate: 5.5e+01/s
    2020-06-18 10:22:33,679 pencil 0/1 INFO :: Building pencil matrix 52/128 (~41%) Elapsed: 1s, 
    Remaining: 1s, Rate: 5.5e+01/s
    2020-06-18 10:22:33,916 pencil 0/1 INFO :: Building pencil matrix 65/128 (~51%) Elapsed: 1s, 
    Remaining: 1s, Rate: 5.5e+01/s
    2020-06-18 10:22:34,167 pencil 0/1 INFO :: Building pencil matrix 78/128 (~61%) Elapsed: 1s, 
    Remaining: 1s, Rate: 5.5e+01/s
    2020-06-18 10:22:34,415 pencil 0/1 INFO :: Building pencil matrix 91/128 (~71%) Elapsed: 2s, 
    Remaining: 1s, Rate: 5.4e+01/s
    2020-06-18 10:22:34,665 pencil 0/1 INFO :: Building pencil matrix 104/128 (~81%) Elapsed: 2s, 
    Remaining: 0s, Rate: 5.4e+01/s
    2020-06-18 10:22:34,915 pencil 0/1 INFO :: Building pencil matrix 117/128 (~91%) Elapsed: 2s, 
    Remaining: 0s, Rate: 5.4e+01/s
    2020-06-18 10:22:35,137 pencil 0/1 INFO :: Building pencil matrix 128/128 (~100%) Elapsed: 2s, 
    Remaining: 0s, Rate: 5.3e+01/s
    2020-06-18 10:22:35,140 __main__ 0/1 INFO :: Solver built
    Traceback (most recent call last):
      File "rayleigh_benard.py", line 83, in <module>
        x, z = domain.all_grids()
    AttributeError: 'Domain' object has no attribute 'all_grids'

```

**Comment by jsoishi** [comment]
Hi @francispoulin, Thanks for the issue report. This occurs because the released (2.1905) version of Dedalus installed by the conda install script is older than the development version you find on github. If you download tests and examples from the master branch on github (which is the default) they may not work.  You can use the following link to get all examples for the current conda release here:

<https://github.com/DedalusProject/dedalus/tree/2.1905/examples>

@kburns Should we open a new issue report to consider installing the current version of scripts as "data" during conda/pip install?

**Comment by francispoulin** [comment]
That's great, thank you for explaining the situation and will make sure to run the right examples.
